sup3r/postprocessing/file_handling.py

```diff
@@ -27,7 +27,8 @@
 H5_ATTRS = {'windspeed': {'scale_factor': 100.0,
                           'units': 'm s-1',
                           'dtype': 'uint16',
-                          'chunks': (2000, 500)},
+                          'chunks': (2000, 500),
+                          'min': 0},
```

Comment:
Probably worth setting a very high max windspeed even if there really isn't one... 120 is what we use for passing ws to SAM.